task_sdk/src/airflow/sdk/execution_time/comms.py

```diff
@@ -131,7 +131,14 @@ class GetVariable(BaseModel):
     type: Literal["GetVariable"] = "GetVariable"
 
 
+class PutVariable(BaseModel):
```

Comment:
Oh, and PutVariable vs SetXCom as the class names for the messages. Should we be consistent between this and #44605?